packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -35,12 +35,21 @@ const CFDsListing = () => {
 
     const accounts_sub_text = is_eu ? localize('Account Information') : localize('Compare accounts');
 
-    const getShortCode = (account: TDetailsOfEachMT5Loginid) => {
-        return account.landing_company_short &&
-            account.landing_company_short !== 'svg' &&
-            account.landing_company_short !== 'bvi'
-            ? account.landing_company_short?.charAt(0).toUpperCase() + account.landing_company_short?.slice(1)
-            : account.landing_company_short?.toUpperCase();
+    const getShortCodeAndRegion = (account: TDetailsOfEachMT5Loginid) => {
+        let short_code_and_region;
+        if (!is_demo && !is_eu_user) {
```

Comment:
no so readable, e.g. `!is_demo` should be `is_real`

Reply:
Sure @matin-binary, I'll add another  computed to the store for `is_real
`